File: trunk/unityClientSample/Assets/src/protocol/MatchRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// match 协议:20
    /// </summary>
	[Proto(value=20,description="match")]
	[ProtoContract]
	public class MatchRequest
	{
        /// <summary>
        ///  select the card group
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string cardGroupId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/MatchRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

@Data
@Proto(value = 20, description = "match")
public class MatchRequest
{
	@Protobuf(description = "select the card group")
	private String cardGroupId;
}
